static/styles/settings.css

```diff
@@ -33,7 +33,7 @@ h3,
     }
 
     .fa-info-circle {
-        top: 1px;
+        left: 1px;
```

Comment:
This change affects all the other places as well where this class is used, so we need to make sure whether all other icons also look fine with this change. Would be better to share before/after screenshots, and if only some icons (or only the one with "Joining the organization" heading) need this change, we can do that instead of changing position of all the icons.

Reply:
Yes, I used a ruler this time, the change of top to left was unnecessary, the top was actually there in all the other same type of icons for alignment, so I left that as it was.

Reply:
@brijsiyag What's the state of the discussion here? Has the PR been updated to make the change more local?  

If not, please post before/after screenshots, as @sahil839 requested. Thanks!

Reply:
hey @alya, I have updated the PR, and screenshots are in first comment of PR, it is ready for review.

Reply:
@sahil839 Does this address your feedback, or is a different approach required (or more screenshots?)? Thanks!